server.js

```diff
@@ -0,0 +1,19 @@
+const http = require('http');
+
+const hostname = '0.0.0.0';
+const port = 3000;
+
+// Computes fibonacci:
```

Comment:
I think using just letter is fine for the iteration, but naming the variable with single letter might not be helpful our code base and the reviewers.